main.go

```diff
@@ -28,6 +29,16 @@ const (
 
 func main() {
 	r := mux.NewRouter()
+
+	r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		page, err := renderer.HTML.Render("home.html", nil)
+		if err != nil {
+			w.WriteHeader(500)
+			w.Write([]byte("Ocorreu um erro ao renderizar a página"))
+			return
+		}
+		w.Write(page)
+	}).Methods("GET")
```

Comment:
Acho que isso não era para ser commitado nesse PR né? Quando for assim, dá uma revisada por conta própria no seu PR antes de mandar, assim você consegue ajustar para que o PR tenha apenas o mínimo de mudanças.